src/infra/nixl_descriptors.cpp

```
@@ -443,6 +443,32 @@ nixlSecDescList::addDescs(nixlSecDescList &&other) {
     addDescs(std::move(other.descs), order::SORTED);
 }
 
+void
+nixlSecDescList::remDescs(std::vector<int> indices, order ord) {
```

Cppcheck (2.20.0)

[style] The function 'remDescs' is never used.

(unusedFunction)